src/OrderImport/Rules/AbstractColissimo.php

```diff
@@ -153,7 +153,16 @@ public function getDescription()
 
     abstract protected function getProductCode(OrderResource $apiOrder);
 
-    abstract protected function getPointId(OrderResource $apiOrder);
+    protected function getPointId(OrderResource $apiOrder)
```

Comment:
It could be great to have the same name between MondialRelais and Colissimo.
getRelayId is better than getPointId